sig-docs/annual-report-2021.md

```diff
@@ -0,0 +1,98 @@
+
+    Yes. We also have `Contributing.md` file.
+  - **All subprojects correctly mapped and listed in [sigs.yaml](/sig-list.md)?**
+
+    Yes.
+      
+  - **What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date?
+    Are you keeping recordings up to date/trends in community members watching recordings?**
+    
+    Quite small (usually 5-10 people) and generally we try to make people feel comfortable to give their opinion.
+    Meeting notes are up to date. We have recording of the meetings uploaded.
+
+
+**How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired?
+Are OWNERS files up to date in these areas?**
+
+<!-- TODO -->
+
+**How does the group get updates, reports, or feedback from Working Groups? Are there any springing up or being retired? 
+Are OWNERS files up to date in these areas?**
+
+<!-- TODO -->
+
+**When was your last monthly community-wide update? (provide link to deck and/or recording)**
+
+<!-- TODO -->
+
+## Membership
+
+**Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
+
+<!-- TODO -->
+
+**How do you measure membership? By mailing list members, OWNERs, or something else?**
+
+We don't have an official way of measuring membership.
+
+**How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?**
+
+PR are generally open for anyone to review. We have [PR wrangler rotation](https://github.com/kubernetes/website/wiki/PR-Wranglers).
+In general each of the approver will taking the rotation. Under exceptional circumstances where there's lots of
+pressure for the wrangler, another approver usually will be willing to help.
+
+For technical reviews, other than SIG Docs Tech Leads, we also notify respective SIGs with necessary knowledge of the
+technical aspect of the content.
+
+**Is there a healthy onboarding and growth path for contributors in your SIG? 
+What are some activities that the group does to encourage this? 
+What programs are you participating in to grow contributors throughout the contributor ladder?**
+
+There is no official onboarding process.
+
+**What programs do you participate in for new contributors?**
+
+<!-- TODO -->
+
+**Does the group have contributors from multiple companies/affiliations? 
+Can end users/companies contribute in some way that they currently are not?**
+
+<!-- TODO -->
+
+## Current initiatives and project health
+
+[ ] Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members.
+
+**What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? 
+Currently underway? What are some of the longer tail projects that your group is working on?**
+
+<!-- TODO -->
+
+**Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?**
+
+<!-- TODO -->
```

Comment:
For this one I don't think we have this, please correct me if I'm wrong

Reply:
Cleanup for that is in progress, it's an ongoing process that doesn't target a specific release.

We could report on PRs linked to the umbrella issue if we want to?

Reply:
I think it may be beneficial to link to the umbrella issue then, do you have the link?

Reply:
https://github.com/kubernetes/website/issues/20232